python/prophet/tests/test_diagnostics.py

```diff
@@ -234,13 +234,13 @@ def test_performance_metrics(self, ts_short, backend):
             metrics=["coverage", "mse"],
         )
         assert set(df_horizon.columns) == {"coverage", "mse", "horizon"}
-        # Skip MAPE
-        df_cv.loc[0, "y"] = 0.0
+        # Handle zero y and yhat and skip MAPE
```

Comment:
You can leave the MAPE skip test as-is and do the test for `0.0` y and `0.0` yhat afterwards.

I guess mdape doesn't have the same problem because it uses the rolling_median function?

Reply:
Updated!

Yep, rolling_median doesn't get rid of the np.nan values, they're still included with the groupby.